common.gypi

```diff
@@ -166,6 +166,11 @@
           ['CXX.host', '$(CXX)'],
         ],
       }],
+      [ 'OS == "linux" and building_nw==1', {
```

Comment:
I tried setting the chromium linker as the default linker but this does not seem to work.